package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "dex-price-estimator",
+  "version": "0.0.1",
+  "description": "A service estimating feasible sell prices for Gnosis Protocol",
+  "main": "dist/index.js",
+  "scripts": {
+    "prebuild": "tslint -c tslint.json -p tsconfig.json --fix",
```

Comment:
tslint is deprecated in favor of eslint
https://github.com/palantir/tslint#tslint